CONTRIBUTORS

```diff
@@ -10,6 +10,7 @@ Iulia-Maria Comșa
 Jan Wassenberg
 Jon Sneyers
 Jyrki Alakuijala
+Khem Raj
```

Comment:
The appropriate file would be [AUTHORS](https://github.com/libjxl/libjxl/blob/main/AUTHORS).

Reply:
done